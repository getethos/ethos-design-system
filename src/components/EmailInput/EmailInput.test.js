import React from 'react'
import { EmailInput } from './index.js'
import renderer from 'react-test-renderer'

describe('EmailInput', () => {
  it('default rendering', () => {
    const tree = renderer
      .create(
        <EmailInput
          name="the-email-input-example"
          allCaps={true}
          labelCopy="Your email"
          data-tid="the-email-input"
          placeholder="example@ethoslife.com"
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
