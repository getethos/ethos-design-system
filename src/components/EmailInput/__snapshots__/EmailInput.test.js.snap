// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailInput default rendering 1`] = `
Array [
  <label
    className="Caption Theinhardt Medium500 GrayPrimary AllCaps"
    htmlFor="the-email-input-example"
  >
    Your email
  </label>,
  <div
    aria-hidden={true}
    className="Spacer"
    style={
      Object {
        "height": 4,
      }
    }
  />,
  <input
    aria-label="the-email-input-example"
    autoComplete="email"
    className="TextInput"
    data-tid="the-email-input"
    name="the-email-input-example"
    onBlur={[Function]}
    onChange={[Function]}
    placeholder="example@ethoslife.com"
    type="email"
    value=""
  />,
  "",
]
`;
